tom_observations/facilities/lco.py

```diff
@@ -259,29 +310,89 @@ def _build_target_fields(self):
+
+                for site in site_selection:
+                    if site in eph_json.keys():
+                        ephemeris_targets[site] = []
+                        ephemeris_windows[site] = []
+                        (mjd_vals, ra_vals, dec_vals, air_vals, sun_alt_vals) = get_radec_ephemeris(eph_json[site],
+                                                                                                    self.cleaned_data['start'],
+                                                                                                    self.cleaned_data['end'],
+                                                                                                    self.cleaned_data['imaging_interval'],
+                                                                                                    'LCO',
+                                                                                                    site)
+                        if mjd_vals is not None:
+                            for i in range(len(ra_vals)-1):
+                                if (air_vals[i] < float(self.cleaned_data['max_airmass']) and
+                                    air_vals[i+1] < float(self.cleaned_data['max_airmass']) and
+                                    air_vals[i] > 1.0 and
+                                    air_vals[i+1] > 1.0 and
+                                    sun_alt_vals[i] < -30.0 and
+                                    sun_alt_vals[i+1] < -30.0):
+
+                                    new_target_fields = {}
+                                    new_target_fields['type'] = 'ICRS'
+                                    new_target_fields['ra'] = (ra_vals[i]+ra_vals[i+1])/2.0
+                                    new_target_fields['dec'] = (dec_vals[i]+dec_vals[i+1])/2.0
+                                    new_target_fields['proper_motion_ra'] = 0.0
+                                    new_target_fields['proper_motion_dec'] = 0.0
+                                    new_target_fields['epoch'] = 2000
+                                    new_target_fields['parallax'] = 0
+
+                                    start = Time(mjd_vals[i], format='mjd')
+                                    end = Time(mjd_vals[i+1], format='mjd')
+
+                                    # store start and end times in the target for a matter of convenience in passing this information forward to the request builder
+                                    new_target_fields['name'] = '{}_{}_{}'.format(target.name, site, i)
+                                    ephemeris_targets[site].append(new_target_fields)
+                                    ephemeris_windows[site].append([start.isot, end.isot])
+                        elif mjd_vals is None and sun_alt_vals==-2:
+                            self.add_error(None, 'Date range outside range available in the provided ephemeris.')
+
+                return (ephemeris_targets, ephemeris_windows)
```

Comment:
You don't need the parentheses around the return values. But it depends on how fussy they are about formatting. Also line 368 should have a space before the ==. Again, how picky are they.